tests/factories.py

```diff
@@ -193,10 +193,26 @@ def update_fulltext(obj, create, extracted, **kwargs):
 
 
 class FilingsFactory(BaseFactory):
+    sub_id = factory.Sequence(lambda n: n)
+
     class Meta:
         model = models.Filings
 
 
+class EFilingsFactory(BaseFactory):
+    file_number = factory.Sequence(lambda n: n)
+
+    class Meta:
+        model = models.EFilings
+
+
+class BaseFilingFactory(BaseFactory):
+    file_number = factory.Sequence(lambda n: n)
+
+class BaseF3PFilingFactory(BaseFilingFactory):
```

Comment:
@jontours:  I completely agree with you on adding more of these, but think it would be okay to shift that work to the issue of adding more tests in general.